tools_docker/test_docker/README.md

```diff
@@ -0,0 +1,45 @@
+# Docker Tests
+
+This is a tool to help test Docker images, for simple cases, but also for complicated tests.
+This allows:
+- automatically running Tracim with Collabora and Elasticsearch on by default (it just needs some tweaks
```

Comment:
- Collabora ? missing version information (like code, desktop, specific version 4.x.x, etc.) 
- Elasticsearch ? same as above

Reply:
docker image name for test are configurable through config file. In case be a bit more explicit about this if you wish.